Saucier720-app/src/app/list/list.component.ts

```diff
@@ -64,11 +64,9 @@ export class ListComponent implements OnInit {
     if (this.newIngredientName && this.newIngredientQuantity > 0) {
       const newIngredient: Ingredient = {
         Name: this.newIngredientName,
-        Quantity: this.newIngredientQuantity,
-        StoreCost: 0, // Example value, replace with actual value if needed
-        OnSale: false, // Example value, replace with actual value if needed
-        SalePrice: 0, // Example value, replace with actual value if needed
-        SaleDetails: '' // Example value, replace with actual value if needed
+        FoodType: '', // Example value, replace with actual value if needed
```

Comment:
Is there a better word than "example value" here, it's not an example as there is nothing there but an empty string

Reply:
ok, i'll change and recommit